pkg/server/handler/authutil.go

```diff
@@ -21,6 +21,7 @@ import (
 	"github.com/skygeario/skygear-server/pkg/server/asset"
 	"github.com/skygeario/skygear-server/pkg/server/plugin/hook"
 	"github.com/skygeario/skygear-server/pkg/server/skydb"
+	"github.com/skygeario/skygear-server/pkg/server/skydb/recordutil"
```

Comment:
I think `recordutil` should exists outside of `skydb` or `handler` package. I’d prefer `pkg/server/util/record` or `pkg/server/recordutil`.